release_docs/RELEASE.txt

```diff
@@ -1129,6 +1129,23 @@ Bug Fixes since HDF5-1.12.0 release
 ===================================
     Library
     -------
+    - Issue #1436 identified a problem with the H5_VERS_RELEASE check in the
+      H5check_version function.
+
+      Investigating the original fix, #812, we discovered some inconsistencies
+      with a new block added to check H5_VERS_RELEASE for incompatibilities.
+      This new block was not using the new warning text dealing with the
+      H5_VERS_RELEASE check and would could cause the warning to be duplicated.
```

Comment:
would could - choose one.

Suggestion for the last sentence in the following paragraph:  "The exception is that an application running with a release version prior to the one with which it was built cannot use new APIs not present in the earlier version. "

Reply:
Not sure that is completely accurate either, closer though.

Reply:
how about "... not present in all release versions"?

Reply:
No, if it is present in the build version and the run version, it doesn't matter that it wasn't in some other release version.  e.g. built with 1.10.9, run with 1.10.7, irrelevant that 1.10.0 - 1.10.6 didn't have the API.

Reply:
But what if the API was removed after it was added, so it was available but then removed and it isn't part of the release API set.